### PR TITLE
[SDEV-1745] feat: add fastem calibrations image to odemis bug reporter

### PR DESCRIPTION
**Pull request by tepals**
The fastem calibrations image, specifically the cell translation plot, gives information about the quality of the multiprobe pattern. It is added to help with a quicker first diagnosis of problems.

### CODE REVIEW

File: src/bugreporter/odemis_bugreporter.py [hunk 1]
```diff
@@ -318,6 +318,17 @@ def compress_files(self):
             if delphi_calib_reps and (time.time() - os.path.getmtime(delphi_calib_reps[-1])) / 3600 < 24:
                 files.append(delphi_calib_reps[-1])
 
+            # Add the latest fastem-calibration image if it's possibly related (ie, less than a week old)
+            fastem_calib_dir = os.path.join(home_dir, "development/fastem-calibrations/images/cell-translation")
```
Comment (pieleric) [inline src/bugreporter/odemis_bugreporter.py:322]: It feels a little odd that the data is stored in `~/development/`. That's a directory that should only be present if developing the software or debugging it. On a normal/stable system, I would assume it should not be there.
Would it be possible to move this directory to `~/fastem-calibration/` (or `~/odemis-status/fastem-calibration/`)? It seems it's just a matter of adjusting the variable in `fastem_calibrations.util` :smile: 

BTW, don't you want to add the whole `~/odemis-status/` directory, while you're at it?

File: src/bugreporter/odemis_bugreporter.py [hunk 2]
```diff
@@ -318,6 +318,23 @@ def compress_files(self):
...
+                calib_file = os.path.join(f, "plot_cell_translations.png")
+                if os.path.isfile(calib_file) and (time.time() - os.path.getmtime(f)) / 3600 < (24 * 7):
+                    files.append(calib_file)
+                    break
+
+            # add the contents of the entire content of odemis-status dir if available,
+            # no need to filter, because the status gets overwritten.
+            odemis_status_dir = os.path.join(home_dir, "odemis-status")
+            if os.path.isdir(odemis_status_dir):
+                files.append(odemis_status_dir)
```
Comment (tepals) [inline src/bugreporter/odemis_bugreporter.py:336]: @pieleric is it save to just add the entire contents of the odemis status dir? I could also filter for anything that ends in `*.txt`

Comment (pieleric) [reply src/bugreporter/odemis_bugreporter.py:336]: I think this is fine. Unless you expect to have large files (after compression). The ticketing system accepts up to 128Mb of attachment... but typically it's nicer if it's below ~20Mb.

**Review by pieleric — APPROVED** [review]

**Review by nandishjpatel — APPROVED** [review]

File: src/bugreporter/odemis_bugreporter.py [hunk 3]
```diff
@@ -318,6 +318,23 @@ def compress_files(self):
             if delphi_calib_reps and (time.time() - os.path.getmtime(delphi_calib_reps[-1])) / 3600 < 24:
                 files.append(delphi_calib_reps[-1])
 
+            # Add the latest fastem-calibration image if it's possibly related (ie, less than a week old)
+            fastem_calib_dir = os.path.join(home_dir, "development/fastem-calibrations/images/cell-translation")
```
Comment (nandishjpatel) [inline src/bugreporter/odemis_bugreporter.py:322]: Does this part have to be updated given this change https://bitbucket.org/delmic/fastem-calibrations/src/ec4b208ca92c5c8f4166b3db280d20b7cce4d82b/src/fastem_calibrations/util.py#lines-65?

Comment (tepals) [reply src/bugreporter/odemis_bugreporter.py]: good catch, fixed it.

**Review by nandishjpatel — APPROVED** [review]

**Pull request merged**